inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/http/DefaultEvent2HttpRequestHandler.java

```diff
@@ -0,0 +1,142 @@
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.slf4j.Logger;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLEncoder;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DefaultEvent2HttpRequestHandler implements IEvent2HttpRequestHandler {
+
+    public static final Logger LOG = InlongLoggerFactory.getLogger(DefaultEvent2HttpRequestHandler.class);
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+    public static final String HTTP_PREFIX = "http://";
+    public static final String HTTPS_PREFIX = "https://";
+    public static final String INLONG_GROUP_ID_HEADER = "inlongGroupId";
+    public static final String INLONG_STREAM_ID_HEADER = "inlongStreamId";
+
+    @Override
+    public HttpRequest parse(HttpSinkContext context, ProfileEvent event)
+            throws URISyntaxException, JsonProcessingException {
+        String uid = event.getUid();
+        HttpIdConfig idConfig = context.getIdConfig(uid);
+        if (idConfig == null) {
+            context.addSendResultMetric(event, context.getTaskName(), false, System.currentTimeMillis());
+            return null;
+        }
+        // get the delimiter
+        String delimiter = idConfig.getSeparator();
+        char cDelimiter = delimiter.charAt(0);
+        // for tab separator
+        byte[] bodyBytes = event.getBody();
+        String strContext = new String(bodyBytes, Charset.defaultCharset());
+        // unescape
+        List<String> columnValues = UnescapeHelper.toFiledList(strContext, cDelimiter);
+        int valueLength = columnValues.size();
+        List<String> fieldList = idConfig.getFieldList();
+        int columnLength = fieldList.size();
+        // get field value
+        Map<String, String> fieldMap = new HashMap<>();
+        for (int i = 0; i < columnLength; ++i) {
+            String fieldName = fieldList.get(i);
+            String fieldValue = i < valueLength ? columnValues.get(i) : "";
+            fieldMap.put(fieldName, fieldValue);
+        }
+
+        // build
+        String uriString = context.getBaseUrl() + idConfig.getPath();
+        if (!uriString.startsWith(HTTP_PREFIX) && !uriString.startsWith(HTTPS_PREFIX)) {
+            uriString = HTTP_PREFIX + uriString;
+        }
+        URI uri = new URI(uriString);
+        String jsonData;
+        HttpUriRequest request;
+        String requestMethod = idConfig.getMethod().toUpperCase();
+        switch (requestMethod) {
+            case "GET":
+                String params = fieldMap.entrySet().stream()
+                        .map(entry -> {
+                            try {
+                                return URLEncoder.encode(entry.getKey() + "=" + entry.getValue(),
```

Comment:
Just need to encode the value, no need for key.

Reply:
fixed.